website/screens/components/select/code/SelectCodePage.tsx

```diff
@@ -253,63 +244,46 @@ const sections = [
         content: (
           <>
             <DxcText as="p">
-              The state of the component is managed internally by the select.
-              Here is an example of how to use an uncontrolled select the submit
-              event.
+              Alternatively you can let the component manage its state
+              internally. The example below shows how to handle the value of an
+              uncontrolled select in the submit event.
             </DxcText>
-            <Example example={uncontrolled} defaultIsVisible />
-          </>
-        ),
-      },
-      {
-        title: "Searchable",
-        content: (
-          <>
             <DxcText as="p">
-              The <Code>searchable</Code> prop enables the search functionality
-              for filtering the select options.
+              The Select's value is empty by default, but an initial,
+              uncontrolled, value can be provided using the{" "}
+              <Code>defaultValue</Code> prop.
             </DxcText>
-            <Example example={searchable} defaultIsVisible />
+            <Example example={uncontrolled} defaultIsVisible />
           </>
         ),
       },
       {
-        title: "Multiple",
+        title: "Error handling",
         content: (
           <>
             <DxcText as="p">
-              The <Code>multiple</Code> prop enables the user to select more
-              than one option. If the component is also optional, it will only
-              add "(Optional)" next to the label and not another option to the
-              listbox.
+              When handling errors, we recommend initializing the{" "}
+              <Code>error</Code> prop with an empty string. This will reserve
+              space for a possible future error message and prevent unintended
+              layout changes. Also, the <Code>onBlur</Code> and{" "}
+              <Code>onChange</Code> events will send <Code>undefined</Code> when
```

Comment:
I think the explanation about the undefined is a bit redundant because it is said already in the table.

Reply:
I do not agree with this at all. I think the explanation of how the errors work is a bit tricky to understand. So, what I'm trying to achieve is another way to explain how they work, combined with an example. I'm not repeating what is said in the prop description at any moment.